example/leafnode-remotes-gateways.yaml

```diff
@@ -0,0 +1,49 @@
+# NOTE: Replace the 1.2.3.4 IPs with real IPs or a domain. Also include a real
+# base64 encoded credentials file.
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: user-credentials
+data:
+  user.ncreds: ...base64-encoded-credentials...
+
+---
+apiVersion: nats.io/v1alpha2
+kind: NatsCluster
+metadata:
+  name: example-nats-cluster
+spec:
+  size: 3
+  version: "2.1.7"
+
+  natsConfig:
+    debug: true
+    trace: true
+
+  pod:
+    volumeMounts:
+    - name: user-credentials
+      mountPath: /etc/nats-creds
+      readOnly: true
+
+  leafnodeConfig:
+    remotes:
+    - url: nats://1.2.3.4:7422
```

Comment:
`url` can be many comma separated, but can also be an array `urls: ["nats://...", "nats://..."]` can be common in case to explicit about the clusters